datahub-web-react/src/app/ingest/source/builder/RecipeForm/common.tsx

```diff
@@ -0,0 +1,420 @@
+export const SCHEMA_DENY: RecipeField = {
+    name: 'schema_pattern.deny',
+    label: 'Deny Patterns',
+    tooltip: 'Use regex here.',
+    type: FieldType.LIST,
+    buttonLabel: 'Add pattern',
+    fieldPath: schemaDenyFieldPath,
+    rules: null,
+    section: 'Schemas',
+    setValueOnRecipeOverride: (recipe: any, values: string[]) =>
+        setListValuesOnRecipe(recipe, values, schemaDenyFieldPath),
+};
+
+const viewAllowFieldPath = 'source.config.view_pattern.allow';
+export const VIEW_ALLOW: RecipeField = {
+    name: 'view_pattern.allow',
+    label: 'Allow Patterns',
+    tooltip: 'Use regex here.',
+    type: FieldType.LIST,
+    buttonLabel: 'Add pattern',
+    fieldPath: viewAllowFieldPath,
+    rules: null,
+    section: 'Views',
+    setValueOnRecipeOverride: (recipe: any, values: string[]) =>
+        setListValuesOnRecipe(recipe, values, viewAllowFieldPath),
+};
+
+const viewDenyFieldPath = 'source.config.view_pattern.deny';
+export const VIEW_DENY: RecipeField = {
+    name: 'view_pattern.deny',
+    label: 'Deny Patterns',
+    tooltip: 'Use regex here.',
+    type: FieldType.LIST,
+    buttonLabel: 'Add pattern',
+    fieldPath: viewDenyFieldPath,
+    rules: null,
+    section: 'Views',
+    setValueOnRecipeOverride: (recipe: any, values: string[]) =>
+        setListValuesOnRecipe(recipe, values, viewDenyFieldPath),
+};
+
+const tableAllowFieldPath = 'source.config.table_pattern.allow';
+export const TABLE_ALLOW: RecipeField = {
+    name: 'table_pattern.allow',
+    label: 'Allow Patterns',
+    tooltip: 'Use regex here.',
+    type: FieldType.LIST,
+    buttonLabel: 'Add pattern',
+    fieldPath: tableAllowFieldPath,
+    rules: null,
+    section: 'Tables',
+    setValueOnRecipeOverride: (recipe: any, values: string[]) =>
+        setListValuesOnRecipe(recipe, values, tableAllowFieldPath),
+};
+
+const tableDenyFieldPath = 'source.config.table_pattern.deny';
+export const TABLE_DENY: RecipeField = {
+    name: 'table_pattern.deny',
+    label: 'Deny Patterns',
+    tooltip: 'Use regex here.',
+    type: FieldType.LIST,
+    buttonLabel: 'Add pattern',
+    fieldPath: tableDenyFieldPath,
+    rules: null,
+    section: 'Tables',
+    setValueOnRecipeOverride: (recipe: any, values: string[]) =>
+        setListValuesOnRecipe(recipe, values, tableDenyFieldPath),
+};
+
+const chartAllowFieldPath = 'source.config.chart_pattern.allow';
+export const CHART_ALLOW: RecipeField = {
+    name: 'chart_pattern.allow',
+    label: 'Allow Patterns',
+    tooltip: 'Use regex here.',
+    type: FieldType.LIST,
+    buttonLabel: 'Add pattern',
+    fieldPath: chartAllowFieldPath,
+    rules: null,
+    section: 'Charts',
+    setValueOnRecipeOverride: (recipe: any, values: string[]) =>
+        setListValuesOnRecipe(recipe, values, chartAllowFieldPath),
+};
+
+const chartDenyFieldPath = 'source.config.chart_pattern.deny';
+export const CHART_DENY: RecipeField = {
+    name: 'chart_pattern.deny',
```

Comment:
I think these Chart fields are specific to Looker, so you can put these in the `looker.ts` file that you created

Reply:
Done!